apps/omg_child_chain/config/dev.exs

```diff
@@ -5,3 +5,9 @@ config :omg_child_chain,
   ethereum_events_check_interval_ms: 500,
   block_queue_eth_height_check_interval_ms: 1_000,
   coordinator_eth_height_check_interval_ms: 1_000
+
+config :omg_child_chain, OMG.ChildChain.Tracer,
```

Comment:
hm... why dev? I think this should be a general setup and should be in `config.exs`

Reply:
Because production only gets deployed as `PROD`. If there's a `DEV` deployment as well developers can write to DD and see the impact of local changes to performance. I'm fine with moving it into `config.exs` if it makes more sense though.

Reply:
Where is this being sent to? How does it integrate?

Reply:
The sole adapter is Datadog (https://github.com/spandex-project/spandex)